cpp/roaring.hh

```diff
@@ -499,6 +499,17 @@ public:
         return api::roaring_bitmap_rank(&roaring, x);
     }
 
+    /**
+     * Returns the index of x in the set.
+     * If the set dosen't contain x , this function will return -1.
+     * The difference with rank function is that this function will return -1
+     * when x isn't in the set, but the rank funciton will return a
+     * non-negative number.
+     */
+    int64_t get_index(uint32_t x) const noexcept {
```

Comment:
@bryceustc nice work. but the name may be a little confusing. `index` start from 0 in general, while the `rank` value start from 1

Reply:
👍 Thanks for your comments, it should start from 0. I have modified the relevant codes and descriptions, please review again when convenient :)